app/controllers/spotlight/catalog_controller.rb

```diff
@@ -201,4 +201,13 @@ def current_page_context
       end
     end
   end
+
+  def additional_export_formats document, format
+    super
+
+    format.solr_json do
+      authorize! :update_solr, @exhibit
```

Comment:
I don't think we want to expose the raw solr document widely, and with the `update_solr` ability someone can come along and grant a service account access to just the solr json.